content/docs/start/experiments/experiment-tracking.md

```diff
@@ -38,9 +38,13 @@ There are some examples below
 from dvclive.lightning import DVCLiveLogger
 
 ...
-
-trainer = Trainer(logger=DVCLiveLogger(save_dvc_exp=True))
-trainer.fit(model)
+with Live(save_dvc_exp=True) as live:
+    trainer = Trainer(
+        logger=DVCLiveLogger(save_dvc_exp=True),
+        default_root_dir="mymodel"
+    )
+    trainer.fit(model)
+    live.log_artifact("mymodel", type="model")
```

Comment:
Not sure this pattern is that helpful for model registry since it will save the full directory of checkpoints. WDYT about something like this:

```
with Live(save_dvc_exp=True) as live:
    checkpoint = pl.callbacks.ModelCheckpoint(dirpath="mymodel")
    trainer = Trainer(
        logger=DVCLiveLogger(save_dvc_exp=True), callbacks=checkpoint
    )
    trainer.fit(model)
    live.log_artifact(checkpoint.best_model_path, type="model")
```

Reply:
If we do this for lightning, we should probably also update the setup for the other tabs so the best model is the one logged instead of the latest

Reply:
> we should probably also update the setup for the other tabs

Not quite get for which tabs exactly. For HF we save best iteration explicitly IIUC, for General Python API this is irrelevant, and for keras I'm also not sure we're saving each iteration. Can you please explain @daavoo ?